.github/workflows/app-artifacts-win.yml

```diff
@@ -41,11 +43,17 @@ jobs:
       uses: crazy-max/ghaction-chocolatey@v1
       with:
         args: install make
+    - name: Azure login
+      if: ${{ inputs.signBinaries }}
+      uses: azure/login@v2
+      with:
+        client-id: ${{ secrets.WINDOWS_CLIENT_ID }}
+        tenant-id: 72f988bf-86f1-41af-91ab-2d7cd011db47
+        subscription-id: ${{ secrets.AZ_SUBSCRIPTION_ID }}
     - name: Fetch certificates
       if: ${{ inputs.signBinaries }}
       shell: pwsh
       run: |
-        az login --service-principal -u ${{ secrets.WINDOWS_CLIENT_ID }} -p ${{ secrets.AZ_LOGIN_PASS }}  --tenant 72f988bf-86f1-41af-91ab-2d7cd011db47
         az keyvault secret download --subscription ${{ secrets.AZ_SUBSCRIPTION_ID }} --vault-name headlamp --name HeadlampAuthCert --file c:\HeadlampAuthCert.pfx --encoding base64
```

Comment:
I see the examples use azure/CLI to run scripts. Is there a need/benefit to doing that with azure/login? https://github.com/marketplace/actions/azure-login#login-with-openid-connect-oidc-recommended

Reply:
azure/cli would allow you to use different azure-cli versions.

azure/login: it might be possible but i dont know how to get federated auth to work without azure/login :(